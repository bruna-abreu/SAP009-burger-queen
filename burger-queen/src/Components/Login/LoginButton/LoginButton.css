.loginBtn {
    display: flex;
    flex-direction: column;
    align-items: center;
}

.loginButton {
    background-color: #FF5800;
    color: white;
    border-radius: 20px;
    width: 60vw;
    height: 6.5vh;
    font-size: 30px;
    margin-top: 2rem;
    position: relative;
    text-align: center;
    font-weight: bold;
    border: none;
}