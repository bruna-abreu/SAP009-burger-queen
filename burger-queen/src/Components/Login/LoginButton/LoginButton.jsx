import './LoginButton.css'

const LoginButton = () => {
    return (
        <section className='loginBtn'>
           <button className="loginButton" aria-label="BotÃ£o de Login">LOGIN</button>  
        </section>
       
    )
}

export default LoginButton