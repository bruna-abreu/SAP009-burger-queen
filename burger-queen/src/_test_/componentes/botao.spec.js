import LoginButton from "../../Components/Login/LoginButton/LoginButton";
import { render, screen } from "@testing-library/react";

it("Deve renderizar o botão corretamente", () => {
    render(
        <LoginButton/>
    )
    const btn = screen.getByText("LOGIN");
    //espera que o botão esteja no documento
    expect(btn).toBeInTheDocument()
    expect(btn).toMatchSnapshot();
})
