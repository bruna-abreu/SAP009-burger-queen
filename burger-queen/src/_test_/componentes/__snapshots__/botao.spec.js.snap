// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deve renderizar o botão corretamente 1`] = `
<button
  aria-label="Botão de Login"
  class="loginButton"
>
  LOGIN
</button>
`;
